x/poolmanager/router.go

```diff
@@ -274,7 +274,7 @@ func (k Keeper) multihopEstimateOutGivenExactAmountInInternal(
 		actualTokenIn := tokenIn
 		// apply taker fee if applicable
 		if applyTakerFee {
-			takerFee, err := k.GetTradingPairTakerFee(ctx, routeStep.TokenOutDenom, tokenIn.Denom)
+			takerFee, err := k.GetTradingPairTakerFee(ctx, tokenIn.Denom, routeStep.TokenOutDenom)
```

Comment:
denom0 and denom1 in these instances make sense because it is from the viewpoint of the pool itself, so there is no direction of what is happening with these denoms.